docarray/document/pydantic_model.py

```diff
@@ -38,7 +38,7 @@ class PydanticDocument(BaseModel):
     text: Optional[str]
     weight: Optional[float]
     uri: Optional[str]
-    tags: Optional[Dict[str, '_StructValueType']]
+    tags: Optional[Dict[str, Any]]
```

Comment:
This however makes `tags` unschema

Reply:
@JoanFM

Reply:
I think is right having tags unschema, it is the free part of Document

Reply:
but the "free-part" still have to be  a valid JSON

Reply:
the right way is to turn on pydantic smart union, as in https://pydantic-docs.helpmanual.io/usage/model_config/#smart-union